account_wallet/tests/common.py

```diff
@@ -23,43 +23,87 @@ class WalletCommon(common.SavepointCase):
     def setUpClass(cls):
         super(WalletCommon, cls).setUpClass()
         load_file(cls.cr, "account_wallet", "tests/data/", "account_wallet_data.xml")
-        cls.wallet_obj = cls.env["account.wallet"]
-        cls.account = cls.env.ref("account_wallet.wallet")
-        cls.partner_account = cls.env["account.account"].search(
-            [("user_type_id.type", "=", "receivable")], limit=1
-        )
+        cls.AccountWallet = cls.env["account.wallet"]
+
+        cls.wallet_account = cls.env.ref("account_wallet.wallet_account")
         cls.wallet_journal = cls.env.ref("account_wallet.wallet_journal")
         cls.wallet_type = cls.env.ref("account_wallet.wallet_type")
-        vals = {
-            "wallet_type_id": cls.wallet_type.id,
-        }
-        cls.wallet = cls.wallet_obj.create(vals)
         cls.partner = cls.env.ref("base.res_partner_2")
 
-    def _provision_wallet(self, amount, wallet=None):
```

Comment:
@legalsylvain I'm not in favor of this in this version as other tests depends on.